docs/Parameters.md

```diff
@@ -31,7 +31,7 @@ Type: string
 
 Possible Values: null
 
-Default: https://raw.githubusercontent.com/Azure/Moodle/master/
+Default: https://raw.githubusercontent.com/Azure/LAMP/master/
```

Comment:
This file should not be manually edited, see notes at top of file. If this was generated by the script then all good.